scripts/qa_crowbarsetup.sh

```diff
@@ -799,8 +799,10 @@ BROADCAST='$net.255'
 EOF
     ifdown br0
     rm -f /etc/sysconfig/network/ifcfg-br0
-    grep -q "^default" /etc/sysconfig/network/routes || \
```

Comment:
I agree, rather revert my change from 386064c4bbada0d4c8170de5f9e9ece546fcb178. I don't know what problem that should solve. It seems useless (unless there are other routes).

Otherwise we should really add a default route if missing, and not overwrite the file.